### PR TITLE
Fix webpack overriding && Add an example with local file dependencies

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: -can't find it-

## What I did
Fixed a few prepublish stuff
Fixed the babelconfig so it loads presets from included 'node_modules'

## How to test
- `npm install`
- `cd examples/test-cra`
- `npm run storybook`

This should result in a successful storybook being launched, and no messages in the terminal about presets not being loaded.

### CODE REVIEW

File: examples/test-cra/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "cra-storybook",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@kadira/storybook": "file:../../packages/react-storybook",
```
Comment (xavcz) [inline examples/test-cra/package.json]: so cool to have direct dependencies on the file system! 💯

File: examples/test-cra/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "@kadira/storybook-addons": "file:../../packages/addons",
+    "@kadira/storybook-channel": "file:../../packages/channel",
+    "@kadira/storybook-ui": "file:../../packages/storybook-ui",
+    "@kadira/storybook-addon-actions": "file:../../packages/addon-actions",
+    "@kadira/storybook-channel-postmsg": "file:../../packages/channel-postmessage",
+    "react-scripts": "0.9.5"
+  },
+  "dependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2"
```
Comment (xavcz) [inline examples/test-cra/package.json]: should we be on `^15.5.4` as the latest version published?

**Review by xavxyz — APPROVED** [review]
🎉  :tada:

File: examples/test-cra/package.json (hunk 1 above)
Comment (ndelangen) [inline examples/test-cra/package.json]: This is to be able to do an actual npm install without linking via lerna.
So when everything is installed flat things should work as expected.

File: examples/test-cra/package.json (hunk 2 above)
Comment (ndelangen) [inline examples/test-cra/package.json]: Yeah for some reason `npm-check-updates` is no longer helping me with this 😞

**Pull request merged**